packages/editor/src/components/rich-text/README.md

```diff
@@ -57,6 +57,34 @@ Render a rich [`contenteditable` input](https://developer.mozilla.org/en-US/docs
 
 *Optional.* A list of autocompleters to use instead of the default.
 
+### `annotations: Array`
+
+*Optional.* A list of annotations that should be applied to the content.
+
+An annotation has the following shape:
+
+```js
+{
+	// The source of the annotation, will be used for the HTML class.
+	"source": "annotations-tester",
+	
+	// The ID of the block to annotate. Currently text annotation will try to annotate all RichText elements within a block.
+	"block": "[blockId]",
```

Comment:
Why not call it `blockId` or `blockClientId` instead of `block`? Initially, I assumed that it will be an object.